core/tests/browse_tests.py

```diff
@@ -0,0 +1,71 @@
+import os
+
+from django.test import TestCase
+from django.db import connection
+from django.conf import settings
+
+from chronam.core.views.browse import get_page_text
+from chronam.core.utils.utils import _get_tip
+from chronam.core.batch_loader import BatchLoader, Batch
+from chronam.core.models import Title, LanguageText
+
+
+class BrowseTests(TestCase):
+    """
+    Tests related to core/views/browse.py
+    """
+    fixtures = ['countries.json', 'languages.json', 'awardee.json']
+
+    def test_full_text_deleted(self):
+        #'sanity' check that 'text' column is removed
+        check = False
+        with connection.cursor() as cursor:
+            cursor.execute("SHOW COLUMNS FROM core_languagetext")
+            rows = cursor.fetchall()
+            for row in rows:
+                if row[0] == 'text':
+                    check = True
```

Comment:
does it make more sense to use `self.fail("core_languagetext.text should not exist. Did you run the migrations?")` here? That would allow you to remove line 21 and 28

Reply:
In a test, yes

Reply:
Changed browse_tests according to the suggestion